City-of-Helsinki/helsinki-paatokset
# PP-355: Add new view that lists hidden decisions.

**To test**
- Checkout branch, run `make drush-cr drush-cim`
- Edit some decisions: https://helsinki-paatokset.docker.so/fi/admin/content/decisions
  - Enable the "Hide decision content" option
- Open the hidden decisions list: https://helsinki-paatokset.docker.so/fi/admin/content/hidden-decisions 
  - The decisions you edited should show up here
- Open the meeting calendar: https://helsinki-paatokset.docker.so/fi/kokouskalenteri (if the page doesn't exist, create it)
  - The meeting time and link font size should now be 16px instead of 18px
- Edit the kaupunginvaltuusto page: https://helsinki-paatokset.docker.so/fi/paattajat/kaupunginvaltuusto
  - Add an image if it doesn't already have one 
  - Make sure it has some sections available so the table of contents shows up
  - Once you save the changes and view the page, there should be a margin between the table to contents and the image. The arrow shouldn't overlap with the image.
- Check out the decisions search page: https://helsinki-paatokset.docker.so/fi/asia/
  - The dropdown hover color should now be black instead of blue
